Framework/DataHandling/test/LoadILLDiffractionTest.h

```diff
@@ -686,6 +693,7 @@ class LoadILLDiffractionTest : public CxxTest::TestSuite {
 
     TS_ASSERT_DELTA(ei, 5.12, 0.01)
     TS_ASSERT_EQUALS(outputWS->run().getProperty("Detector.calibration_file")->value(), "none")
+    TS_ASSERT_EQUALS(runList->value(), std::string("104"))
```

Comment:
Do you need `std::string`?

Reply:
A string literal would be sufficient, actually, you are correct.